lsp-common.el

```diff
@@ -36,6 +36,9 @@
                                (_ "file://"))
   "Prefix for a file-uri.")
 
+(defvar-local lsp-buffer-uri nil
```

Comment:
Alternatively, is it possible to use a programmer-provided function for computing the current buffer uri too?

Reply:
It is possible, I choose this approach since it allows inlining and the original method was inlined.